### ISSUE TITLE
Got problems in training

### ISSUE THREAD

**Issue by TommyTang930** [MaybeShewill-CV/lanenet-lane-detection#305]
**1、Environment configuration:**

tensorboard                        1.13.1   
tensorflow-estimator           1.13.0   
tensorflow-gpu                    1.13.1 
numpy                                 1.15.1
opencv-python                    4.1.1.26 

 **２、input command :** 

python tools/train_lanenet.py --net vgg --dataset_dir ./data/training_data_example/ -m 1

**3、output :** 

---multi GPU---
---tfrecords_file_paths--- 
['./data/training_data_example/tfrecords/train_2000_3000.tfrecords', './data/training_data_example/tfrecords/train_1000_2000.tfrecords', './data/training_data_example/tfrecords/train_3000_3082.tfrecords', './data/training_data_example/tfrecords/train_0_1000.tfrecords']
W1016 08:55:46.904392 12227 deprecation.py:323] From /home/guoliangtang/anaconda3/lib/python3.7/site-packages/tensorflow/python/ops/control_flow_ops.py:3632: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
---tfrecords_file_paths--- ['./data/training_data_example/tfrecords/val_0_181.tfrecords']
W1016 08:55:47.003061 12227 deprecation.py:323] From /home/guoliangtang/anaconda3/lib/python3.7/site-packages/tensorflow/python/training/learning_rate_decay_v2.py:321: div (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Deprecated in favor of operator or tf.math.divide.
---GPU Num--- 2

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

Traceback (most recent call last):
  File "tools/train_lanenet.py", line 717, in <module>
    train_lanenet_multi_gpu(args.dataset_dir, args.weights_path, net_flag=args.net_flag)
  File "tools/train_lanenet.py", line 556, in train_lanenet_multi_gpu
    train_images, train_binary_labels, train_instance_labels, train_net, optimizer
  File "tools/train_lanenet.py", line 201, in compute_net_gradients
    instance_label=gt_instance_labels, name='lanenet_model'
  File "/home/guoliangtang/Desktop/LaneNet/lanenet-lane-detection-master/lanenet_model/lanenet.py", line 82, in compute_loss
    reuse=self._reuse
  File "/home/guoliangtang/Desktop/LaneNet/lanenet-lane-detection-master/lanenet_model/lanenet_front_end.py", line 43, in build_model
    reuse=reuse
  File "/home/guoliangtang/Desktop/LaneNet/lanenet-lane-detection-master/semantic_segmentation_zoo/vgg16_based_fcn.py", line 357, in build_model
    self._vgg16_fcn_encode(input_tensor=input_tensor, name='vgg16_encode_module')
  File "/home/guoliangtang/Desktop/LaneNet/lanenet-lane-detection-master/semantic_segmentation_zoo/vgg16_based_fcn.py", line 137, in _vgg16_fcn_encode
    need_layer_norm=True
  File "/home/guoliangtang/Desktop/LaneNet/lanenet-lane-detection-master/semantic_segmentation_zoo/vgg16_based_fcn.py", line 67, in _vgg16_conv_stage
    bn = self.layerbn(inputdata=conv, is_training=self._is_training, name='bn')
  File "/home/guoliangtang/Desktop/LaneNet/lanenet-lane-detection-master/semantic_segmentation_zoo/cnn_basenet.py", line 316, in layerbn
    return tf.keras.layers.batch_normalization(inputs=inputdata, training=is_training, name=name)
AttributeError: module 'tensorflow._api.v1.keras.layers' has no attribute 'batch_normalization'

**Comment by MaybeShewill-CV** [comment]
@TommyTang930 Seems Tensorflow 1.13 are not correctly installed:)

**Comment by TommyTang930** [comment]
Thank you for your advice. It finally worked.  I have tried to install different tensorflow versions. Finally, I got the right one Tensorflow-gpu=1.14.0, tensorboard=1.14.0, tensorflow-estimator. Also, Numpy version is also very important. I tried Numpy=1.17.2, It finally worked. By the way, I run it on two GPUs (GeForce RTX 2080). Thanks a lot again. I am trying to test different networks on LaneNet.

**Comment by TommyTang930** [comment]
@MaybeShewill-CV 
Thank you for your advice. It finally worked.  I have tried to install different tensorflow versions. Finally, I got the right one Tensorflow-gpu=1.14.0, tensorboard=1.14.0, tensorflow-estimator. Also, Numpy version is also very important. I tried Numpy=1.17.2, It finally worked. By the way, I run it on two GPUs (GeForce RTX 2080). Thanks a lot again. I am trying to test different networks on LaneNet.
